### PR TITLE
chore: add description for driver

### PR DESCRIPTION
**Pull request by nliu-ms**

### CODE REVIEW

**Review by jayzhang — APPROVED** [review]

File: packages/fx-core/resource/package.nls.json [hunk 1]
```diff
@@ -1143,6 +1143,11 @@
   "driver.m365Bot.error.invalidParameter": "Following parameter is missing or invalid for %s action: %s.",
   "driver.m365Bot.error.unhandledError": "Unhandled error happened in %s action: %s",
   "driver.teamsApp.description.copyAppPackageForSPFxDriver": "copies the generated Teams app package to SPFx solution.",
+  "driver.teamsApp.description.createDriver": "Create Teams app.",
```
Comment (jidddddd) [inline packages/fx-core/resource/package.nls.json:1146]: create a Teams app.

File: packages/fx-core/resource/package.nls.json [hunk 2]
```diff
@@ -1143,6 +1143,11 @@
...
+  "driver.teamsApp.description.updateDriver": "Update Teams app.",
```
Comment (jidddddd) [inline packages/fx-core/resource/package.nls.json:1147]: update a Teams app.

File: packages/fx-core/resource/package.nls.json [hunk 3]
```diff
@@ -1143,6 +1143,11 @@
...
+  "driver.teamsApp.description.publishDriver": "Publish Teams app to tenant app catalog.",
```
Comment (jidddddd) [inline packages/fx-core/resource/package.nls.json:1148]: publish a Teams app to tenant app catalog.

File: packages/fx-core/resource/package.nls.json [hunk 4]
```diff
@@ -1143,6 +1143,11 @@
...
+  "driver.teamsApp.description.validateDriver": "Validate Teams app.",
```
Comment (jidddddd) [inline packages/fx-core/resource/package.nls.json:1149]: validate a Teams app.

**Pull request merged**